examples/compiled/repeat_histogram.vg.json

```diff
@@ -20,10 +20,6 @@
                     "as": "Horsepower"
                 },
                 {
-                    "type": "filter",
```

Comment:
Looking at this one confirms that bin does not handle null filter well, treating them as zero.  (Look at the bin for horsepower.)